blobfuse/src/BlobfuseGcCache.cpp

```diff
@@ -19,6 +19,18 @@ bool gc_cache::check_disk_space()
     double available;
     unsigned long long used;
 
+    static bool user_detected = false;
+    static bool is_sudo_user = false;
+
+    if (!user_detected) {
+        // If effective uid = 0 then we are running as sudo
+        if (geteuid() == 0)
+            is_sudo_user = true;
+        user_detected = true;
+
+        AZS_DEBUGLOGV("GC_Cache User identified. Privillaged : %d", (int)(is_sudo_user));
+    }
+
     if (config_options.cacheSize == 0) {
         struct statvfs buf;
         if(statvfs(cache_folder_path.c_str(), &buf) != 0)
```

Comment:
Also log  an error if this stavfs lookup fails

Reply:
Failure log added with errno.